bsip-0071.md

```diff
@@ -9,23 +10,43 @@
 
 # Abstract
 
-This BSIP proposes a new solution to handle bad debt: the core idea is, while bad debt appears, the system does not take over the bad debt positions; instead, it accepts the smartcoin devaluation caused by bad debt and it lets borrowing, margin call, and force settlement all operate referring to the BTS/devaluated smartcoin price.  This solution avoids any global/partial settlement, and keeps the borrowing, margin calls and force settlement features to continue and allow the market to decide how to finally remove bad debt, either by BTS price restoration or by debt position adjustment/ margin call order filling/force settlement.  
+This BSIP proposes a new solution to handle bad debt: the core idea is, while bad debt appears, 
+the system does not take over the bad debt positions; 
+instead, it accepts the smartcoin devaluation caused by bad debt and it lets borrowing, margin call, 
+and force settlement all operate referring to the BTS/devaluated smartcoin price. 
+This solution avoids any global/partial settlement, and keeps the borrowing, margin calls and force settlement features 
+to continue and allow the market to decide how to finally remove bad debt, 
+either by BTS price restoration or by debt position adjustment/ margin call order filling/force settlement. 
+In addition, this BSIP allows the asset owner to define an optional threshold price to prevent external market manipulation.
 
 # Motivation
 
-BitShares has a mechanism to handle debt positions whose collateral is valued less than the debt itself ("bad debt").  This mechanism is called global settlement ("black swan") and is triggered when the published feed price (FP) is less than or equal to the product of the global settlement price (P<sub>gs</sub>) and the maximum short-squeeze ratio (MSSR).
+BitShares has a mechanism to handle debt positions whose collateral is valued less than the debt itself ("bad debt"). 
+This mechanism is called global settlement ("black swan") and is triggered when the published feed price (FP) is less 
+than or equal to the product of the global settlement price (P<sub>gs</sub>) and the maximum short-squeeze ratio (MSSR).
 
 FP &le; P<sub>gs</sub> &times; MSSR
 
 If this does occur it means that the collateral ratio (CR) of at least one debt position is less than the MSSR.
 
-Global settlement is not a good way to handle bad debt, as can be seen to what happened to bitUSD.  After global settlement was triggered for bitUSD in December 2018:
+Global settlement is not a good way to handle bad debt, as can be seen to what happened to bitUSD.  
+After global settlement was triggered for bitUSD in December 2018:
 
 - traders could no longer borrow bitUSD;
 - the price of bitUSD dropped below USD because of insufficient collateral; and,
 - it will take a long time for bitUSD to be revived.
```

Comment:
bitUSD was in a GS state from Nov to Feb? BSIP76 has been active for almost a month now and will persist until the price recovers (same conditions for GS recovery - may be a long time).

Reply:
Revival would be in effect only after reaching MCR of the equivalent necessary value.